sci-biology/abyss/metadata.xml

```diff
@@ -10,7 +10,12 @@
     <name>Gentoo Haskell</name>
   </maintainer>
   <use>
+    <flag name="boost">use boost libraries</flag>
+    <flag name="bwa">if you want to run abyss-bwa mapper instead of its own abyss-map</flag>
+    <flag name="mpich">use mpich implementation instead of openmpi</flag>
+    <flag name="mpi">use openmpi</flag>
     <flag name="misc-haskell">build abyss-samtobreak tool, pull in haskell toolchain</flag>
+    <flag name="sparsehash">use Google sparsehash (highly recommended)</flag>
```

Comment:
I wanted to keep it optional, as upstream did. The tool should be slow without that, per upstream.

Reply:
Here is a snippet from configure.log if sparsehash is not explicitly enabled:

```
checking boost/unordered_set.hpp usability... yes
checking boost/unordered_set.hpp presence... yes
checking for boost/unordered_set.hpp... yes
checking sqlite3.h usability... yes
checking sqlite3.h presence... yes
checking for sqlite3.h... yes
checking for main in -lsqlite3... yes
checking for x86_64-pc-linux-gnu-g++ option to support OpenMP... -fopenmp
configure: error: ABySS should be compiled with Google sparsehash to
	reduce memory usage. It may be downloaded here:
	https://code.google.com/p/sparsehash/

	If you do not wish to use sparsehash, specify --without-sparsehash.
```

Reply:
@mmokrejs the point there was to enable it by default via `+sparsehash`, not making it unconditionally.